### PR TITLE
Added how we do PRs doc

### PR DESCRIPTION
**Pull request by dominikwilkowski**
Please let me know if I missed something. This is just the start of a doc that we can all contribute to as we go along.

### CODE REVIEW

**Comment by sawks** [comment]
under "Merging PRs" we could change "the developer" to "the author" for clarity?

**Comment by dominikwilkowski** [comment]
Great point @sawks. Fixed

File: HOW-WE-DO-PRS.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# How we do PRs
+
+> This document outlines the procedures for shipping PRs, including the who/what/where of PR reviews.
+
+## General principals
```
Comment (elisechant) [inline HOW-WE-DO-PRS.md]: typo 👉  principles

**Review by elisechant — APPROVED** [review]
Awesome.

One typo, but Approve otherwise.

File: HOW-WE-DO-PRS.md (hunk 1 above)
Comment (dominikwilkowski) [reply HOW-WE-DO-PRS.md]: fixed

**Pull request merged**